__init__.py

```diff
@@ -57,21 +64,68 @@ def handle_query_name(self, message):
         device = DeviceApi().get()
         self.speak_dialog("my.name.is", data={"name": device["name"]})
 
+    @intent_file_handler("EnablePreciseDev.intent")
+    @on_error_speak_dialog('must.update')
+    def handle_use_precise_dev(self, message):
+        from mycroft.configuration.config import (
+            LocalConf, USER_CONFIG, Configuration
+        )
+
+        wake_word = Configuration.get()['listener']['wake_word']
+
+        new_config = {
+            'precise': {
+                "dist_url": self.PRECISE_DEV_DIST_URL,
+                "model_url": self.PRECISE_DEV_MODEL_URL
+            },
+            'hotwords': {wake_word: {'module': 'precise', 'sensitivity': 0.5}}
+        }
+        user_config = LocalConf(USER_CONFIG)
+        user_config.merge(new_config)
+        user_config.store()
+
+        self.bus.emit(Message('configuration.updated'))
+        self.speak_dialog('precise.devmode.enabled')
+
+    @intent_file_handler("DisablePreciseDev.intent")
+    @on_error_speak_dialog('must.update')
+    def handle_use_precise_dev(self, message):
+        from mycroft.configuration.config import (
+            LocalConf, USER_CONFIG
+        )
+
+        for item in glob(expanduser('~/.mycroft/precise/precise-engine*')):
+            self.log.info('Removing: {}...'.format(item))
+            rmtree(item)
```

Comment:
I had an issue here the glob includes a precise-engine-0.2.0.tar.gz file in the same directory as the precise-engine folder and fails asking me to update my software. (rmtree can't delete files)

Is this error correct? Is the intention that the tar.gz is included? or should something like:
```python
            if isdir(item):
                rmtree(item)
            else:
                os.remove(item)
```

be used?

Reply:
Good catch! Yeah, deleting all the files/folders is the intention. Fixed.